src/helpers/translations/translations.js

```diff
@@ -1,20 +1,21 @@
 export default {
   locale: "en",
   crypto: {
+    bluetext: "Always for sale",
     ethereum: "ethereum based",
     tokens: "tokens",
     subHeading:
-      "Let your digital assets make a valuable contribution to the world.",
+      "Link your social profile to your preferred public address and verifiably showcase your patronage on our platform.",
     nft: "Non-Fungible Token",
     whatIsANFT:
       " is a way to ensure digital assets are unique and easily tradable on a blockchain.",
     harbergerTax: "Harberger Tax"
   },
   muggle: {
-    ethereum: "digital",
-    tokens: "animals",
-    subHeading:
-      "Become the guardian of a wildcard and contribute to endangered animals",
+    bluetext: "",
+    ethereum: "Raising funds for",
```

Comment:
```suggestion
    ethereum: "raising funds for",
```
(Only if you accept the change from the bluetext).